src/mono/wasm/host/BrowserArguments.cs

```diff
@@ -36,9 +36,4 @@ public void ParseJsonProperties(IDictionary<string, JsonElement>? properties)
         if (properties?.TryGetValue("forward-console", out JsonElement forwardConsoleElement) == true)
             ForwardConsoleOutput = forwardConsoleElement.GetBoolean();
     }
-
-    public void Validate()
-    {
-        CommonConfiguration.CheckPathOrInAppPath(CommonConfig.AppPath, HTMLPath, "html-path");
```

Comment:
If this is not null, then would it make sense to still check the path?
Is there a good way to check if the project is using wasm-sdk or not? And if it's not then check this?

Reply:
The thing is StaticWebAssets doesn't copy assets to output until you publish the app. Instead they use the generated manifest and serve files from several folders. So the path can be set from WasmSDK project, but the file would not be there (we would have to read the SWA manifest and check correct source folder, but I think it's not worth it).

We can differentiate between old-style and WasmSDK project by passing argument to the WasmAppHost, but my original approach was just to check for SWA manifest. 

Do you think it would be better to have extra arg to clearly switch between project styles? Also note that we will (not sure if in .NET 8) migrate the console template to WasmSDK

Reply:
> Do you think it would be better to have extra arg to clearly switch between project styles? Also note that we will (not sure if in .NET 8) migrate the console template to WasmSDK

Since we construct the arguments passed to `WasmAppHost`, we can add a parameter for this. This would also allow us to cleanly choose which server to use, whenever we migrate the templates.